pkg/model/model.go

```diff
@@ -137,25 +137,45 @@ func (m *Model) GetCRDs() ([]*CRD, error) {
 				// It's a Status field...
 				continue
 			}
-			if fieldConfig.From == nil {
-				// Isn't an additional Spec field...
-				continue
-			}
-			from := fieldConfig.From
-			memberShapeRef, found := m.SDKAPI.GetInputShapeRef(
-				from.Operation, from.Path,
-			)
-			if found {
-				memberNames := names.New(targetFieldName)
-				crd.AddSpecField(memberNames, memberShapeRef)
-			} else {
-				// This is a compile-time failure, just bomb out...
-				msg := fmt.Sprintf(
-					"unknown additional Spec field with Op: %s and Path: %s",
+
+			var found bool
+			var memberShapeRef *awssdkmodel.ShapeRef
+
+			if fieldConfig.From != nil {
+				from := fieldConfig.From
+				memberShapeRef, found = m.SDKAPI.GetInputShapeRef(
 					from.Operation, from.Path,
 				)
-				panic(msg)
+				if !found {
+					// This is a compile-time failure, just bomb out...
+					msg := fmt.Sprintf(
+						"unknown additional Spec field with Op: %s and Path: %s",
+						from.Operation, from.Path,
+					)
+					panic(msg)
+				}
+			} else if fieldConfig.CustomField != nil {
```

Comment:
It seems like the custom field is being added to both Spec and Status-- is that intended? Am I misreading?